docs/book/src/clusteropenstack/configuration.md

```diff
@@ -69,10 +69,21 @@ We currently require at least OpenStack Pike.
 
 ## Operating system image
 
-We currently depend on an up-to-date version of cloud-init otherwise the operating system choice is yours. The kubeadm bootstrap provider we're using also depends on some pre-installed software like a container runtime, kubelet, kubeadm, etc.. . For an examples how to build such an image take a look at [image-builder (openstack)](https://image-builder.sigs.k8s.io/capi/providers/openstack.html).
+### Cloud-init based images
+
+We currently depend on an up-to-date version of cloud-init, otherwise the operating system choice is yours. The kubeadm bootstrap provider we're using also depends on some pre-installed software like a container runtime, kubelet, kubeadm, etc.. . For an examples how to build such an image take a look at [image-builder (openstack)](https://image-builder.sigs.k8s.io/capi/providers/openstack.html).
 
 The image can be referenced by exposing it as an environment variable `OPENSTACK_IMAGE_NAME`.
 
+### Ignition based images
+
+Some OS like [Fedora CoreOS](https://getfedora.org/en/coreos) or [Flatcar](https://www.flatcar.org/) do not use Cloud-init but [Ignition](https://coreos.github.io/ignition/) to provision the instance. You need to export the [Ignition experimental feature](https://cluster-api.sigs.k8s.io/tasks/experimental-features/ignition.html): `export EXP_KUBEADM_BOOTSTRAP_FORMAT_IGNITION=true`
+
+To use Flatcar image:
+* Build the image with the [image-builder](https://image-builder.sigs.k8s.io/capi/providers/openstack.html): `make OEM_ID=openstack build-qemu-flatcar`
+* Export the name of the uploaded image: `export OPENSTACK_FLATCAR_IMAGE_NAME=flatcar-stable-3374.2.5-kube-v1.25.6`
+* Use one of the following flavor: `flatcar` or `external-cloud-provider-flatcar` (_NOTE_: If you select `external-cloud-provider-flatcar`, don't forget to refer to the [external-cloud-provider](https://cluster-api-openstack.sigs.k8s.io/topics/external-cloud-provider.html) section)
```

Comment:
I'm getting confused what kind of flavor is that? OpenStack flavort?

Reply:
this will be very confusing then, every openstacker knows what's the meaning of flavor in openstack world
so better to add those hint you mentioned above in the doc as well

Reply:
I like the suggested change as it disambiguates the usage of `flavor` here.

Reply:
Much better, thank you!